include/tdl/ParamValue.h

```diff
@@ -1138,7 +1138,7 @@ namespace tdl
     std::string s;
     if (full_precision)
     {
-      os.precision(15);
+      os.precision(std::numeric_limits<double>::max_digits10);
```

Comment:
```suggestion
      os.precision(std::numeric_limits<double>::max_digits10 - 1);
```

Maybe?

https://godbolt.org/z/a68b9jEo5

Reply:
In deinem Beispiel kommt als output:
```
   max_digits10: 7.6000000000000005
submax_digits10: 7.600000000000001

   max_digits10: 7.6000000000000014
submax_digits10: 7.600000000000001
```
Thats exactly what I want to avoid. different numbers should not be rounded to the same values. Did I miss anything?
I noticed, my explanation above is wrong, it has nothing to do with the decimial point'.' or the preceding sign  '-'.

Reply:
Ah, then you need `max_digits10`. But you'll need to adapt the tests since `7.6` is represented by `7.5999999999999996`

Reply:
Now I understand. I haven't looked at what the test complains about. 😅
I just saw the line and knew it wasn't what is required.